randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1416,12 +1445,16 @@ Extra - room_id: [40]
   * Extra - door_idx: (101,)
   > Door to Sanctuary Cache
       All of the following:
+          Morph Ball
           Any of the following:
-              Charge Beam
-              Gravity Suit and Screw Attack
-          Any of the following:
-              Wave Beam
-              Missiles ≥ 2 and Can Use Power Bombs
+              # To hurt the golden scisers
+              Missiles ≥ 5
+              All of the following:
+                  Charge Beam
+                  Missiles ≥ 2 or Wave Beam
+              Gravity Suit and Missiles ≥ 2 and Screw Attack
+          # To hurt the scisers in the enclosed area
+          Wave Beam or Can Use Power Bombs
```

Comment:
Does this wave beam requirement also need charge beam?

Reply:
Regular scisers are hurt by non-charged shots

Reply:
ahh they are regular after reforming